subprojects/emcee/worker/src/main/kotlin/com/avito/emcee/worker/StartWorkerCommand.kt

```diff
@@ -0,0 +1,53 @@
+package com.avito.emcee.worker
+
+import com.avito.emcee.worker.WorkerQueueApi.Companion.create
+import com.avito.emcee.worker.internal.SingleInstanceAndroidDeviceTestExecutorProvider
+import com.avito.emcee.worker.internal.TestJobConsumerImpl
+import com.avito.emcee.worker.internal.TestJobProducerImpl
+import kotlinx.cli.ArgType
+import kotlinx.cli.ExperimentalCli
+import kotlinx.cli.Subcommand
+import kotlinx.cli.required
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.runBlocking
+import retrofit2.Retrofit
+import kotlin.time.ExperimentalTime
+
+@ExperimentalTime
+@ExperimentalCli
+internal class StartWorkerCommand(
+    name: String,
+    description: String
+) : Subcommand(name, description) {
+
+    private val workerId: String by option(
+        type = ArgType.String,
+        description = "Help Queue to identify worker's interactions"
```

Comment:
```suggestion
        description = "Helps Queue to identify worker's interactions"
```
Is it a statement or request?